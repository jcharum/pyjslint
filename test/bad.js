//
// This is code to exercise all of the reporting functions in
// pyjslint.
//

var foo = function() {
    var unused; // unused variable;
    global = 5; // global variable
} // missing ;

