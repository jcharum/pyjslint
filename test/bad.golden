test/bad.js 6: Expected exactly one space between 'function' and '('.
var foo = function() {

test/bad.js 7: Expected 'var' at column 1, not column 5.
    var unused; // unused variable;

test/bad.js 8: Expected 'global' at column 1, not column 5.
    global = 5; // global variable

test/bad.js 9: Expected ';' and instead saw '(end)'.
} // missing ;

test/bad.js 8: implied global 'global'

test/bad.js 6: unused variable 'unused'

